src/pages/index.js

```diff
@@ -7,6 +7,7 @@ import 'bootstrap/dist/css/bootstrap.min.css';
 import { library } from '@fortawesome/fontawesome-svg-core'
 import MainContent from "../components/main-content";
 import AboutUs from "../components/aboutus";
+import Testimonial from "../components/Testimonial";
```

Comment:
@faithghlee : This should be, `import Testimonial from "../components/testimonial";` which is why there's a build error. 😄